language_formatters_pre_commit_hooks/pretty_format_java.py

```diff
@@ -65,6 +66,12 @@ def pretty_format_java(argv: typing.Optional[typing.List[str]] = None) -> int:
         default=None,
         help="Path to Google Java Formatter jar file. Will be downloaded if not defined. Note that --google-java-formatter-version will be ignored if this parameter is defined (default: %(default)).",
     )
+    parser.add_argument(
+        "--google-java-formatter-jar-checksum",
```

Comment:
nit. To keep argument uniform with the equivalent for kotlin, do you have objections on naming the argument as `--formatter-jar-checksum` in here as well?

Reply:
I can rename it. I chose this name to match `--google-java-formatter-version` but no strong preferences